### PR TITLE
R4R: Gaia 9000

### PR DESCRIPTION
**Pull request by zmanian**
Gaia-9000 genesis

### CODE REVIEW

File: gaia-9000/gaia9000scripts/.vscode/debug-ts.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+'use strict';
```
Comment (cwgoes) [inline gaia-9000/gaia9000scripts/.vscode/debug-ts.js]: What are all these `.vscode` files?

**Comment by alexanderbez** [comment]
Will the bech32 prefixes be updated?

**Comment by zmanian** [comment]
the prefixes are updated in the `genesis.json` and I committed the scripts I used for updating the.

Re the `.vscode` files: They were part of the boiler plate generator for typecript I've been using. I think there is a case for leaving them in there.

File: gaia-9000/genesis.json [hunk 2]
```diff
@@ -0,0 +1,7649 @@
...
+            }
+        ]
+    },
+    {
+        "address": "cosmos12u2e2qn74zlk34g223l7jthx53t2wgcj8g5xjx",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+
```
Comment (zmanian) [inline gaia-9000/genesis.json]: @cwgoes Do this look like a correct way to configure proposer rewards?
I'm not clear on how proposer rewards and overall inflation are related

**Comment by cwgoes** [comment]
> Re the .vscode files: They were part of the boiler plate generator for typecript I've been using. I think there is a case for leaving them in there.

OK, no big deal.

**Comment by cwgoes** [comment]
@zmanian A few notes:

- Unbonding time is still three weeks - intentional?
- Inflation is set to 1000% annually - that looks OK to me - although note that it's still just processed once per hour
- I don't think you can set base/bonus proposer rewards above 100%, it will probably cause panics (or we would end up with a negative fee pool...), also cc @rigelrozanski 

Have you tried starting `gaiad` with this file on the four starting validators?

File: gaia-9000/genesis.json (hunk 2 above)
Comment (rigelrozanski) [reply gaia-9000/genesis.json]: you have currently set this to 10000% hahaha - 

The default values are here:
https://github.com/cosmos/cosmos-sdk/blob/bfc7944b55382c0749fecf294289e3b65792c4dc/x/distribution/types/genesis.go#L42

for your understanding `sdk.NewDecWithPrec(1, 2)` is equivalent to `0.01` which is equivalent to saying "hay that value is 1%"

**Comment by zmanian** [comment]
I'm trying to understanding how to get a ~3 day unbonding period.

**Pull request merged**